mods/default/nodes.lua

```diff
@@ -2328,29 +2328,31 @@ function default.chest.register_chest(name, d)
 	def_closed.tiles[5] = def.tiles[3] -- drawtype to make them match the mesh
 	def_closed.tiles[3] = def.tiles[3].."^[transformFX"
 
-	minetest.register_node("default:" .. name, def_closed)
-	minetest.register_node("default:" .. name .. "_open", def_opened)
+	minetest.register_node(name, def_closed)
```

Comment:
Same as doors: Add the registration-specific overwrite character `:` in front, which ensures that `name` is the plain name (ensures correct drop and swap node name values).

Reply:
If anyone need to override, he need to add ":" in external mod to the "name". Do not like to force all overrides by hard-coding ":" (The sense of ":" gets lost). 
But agree I see the issue with drops and other places, that needs thee proper name without ":"

Reply:
Solved: The "original" name (prefixed_name) is passed from register_chest to both register_node. For all other things in definition the prefixless (with removed ":")  "name" is used